exporter/collector/integrationtest/cmd/recordfixtures/main.go

```diff
@@ -45,25 +46,66 @@ func (t *FakeTesting) Helper()      {}
 func (t *FakeTesting) Name() string { return "record fixtures" }
 
 func main() {
-	t := &FakeTesting{}
 	ctx := context.Background()
 	endTime := time.Now()
 	startTime := endTime.Add(-time.Second)
+	t := &FakeTesting{}
+
+	for _, arg := range os.Args[1:] {
+		switch arg {
+		case "logs":
+			logs(ctx, t, endTime)
```

Comment:
Is there a reason why we only want to record one of them?  Should we just record both metrics and logs when this is run?

Reply:
Good point